src/notes.ts

```diff
@@ -99,11 +101,12 @@ chrome.storage.local.get([
   // Notes
   state.notes = notes;
   const activeFromUrl = window.location.search.startsWith("?") && decodeURIComponent(window.location.search.substring(1)); // Bookmark
-  const activeCandidate = activeFromUrl || lastActive || "Clipboard";
-  state.active = (activeCandidate in notes) ? activeCandidate : null;
-  if (state.active && !activeFromUrl) {
-    notesHistory.replace(activeCandidate);
+  const activeCandidates = [activeFromUrl, lastActive, Object.keys(notes).sort()[0]]; // ordered by importance
```

Comment:
Determining which note to open is based on this preference:

1. The highest priority has the URL (like a bookmark). **Example:** URL ending with `?Todo` would open "Todo" note.
2. Last visited note would next to consider.
3. If the active note is deleted, the first available note is selected.

If none of the above can be applied, like when user deleted all notes, no note is open.